scripts/reports/day_report_cronjob.yaml

```diff
@@ -1,17 +1,24 @@
-apiVersion: batch/v1
+apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
   name: dayreport
 spec:
-  schedule: "0 0 * * *"
+  schedule: "0 0 * * *"  #  At 00:00
```

Comment:
If it's more obvious you can write `schedule: @midnight` instead (see "[Cron schedule syntax](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax)").

Reply:
Sure. Right now we are generating at midnight in the curator cluster. Once we confirm that whether it will always be generated at midnight or will have preference according to the user, we will make it midnight.

Reply:
I choose to use 00:00 instead of midnight because we want to run it at 00:00 **UTC**. Using midnight may be confusing.